lua/core/terminal.lua

```diff
@@ -3,6 +3,7 @@ local utils = require "utils"
 
 M.config = function()
   lvim.builtin["terminal"] = {
```

Comment:
can we make this to `lvim.builtin.terminal` to make it look like the rest of them